pkg/cassandra/config/config.go

```diff
@@ -44,7 +44,6 @@ type Configuration struct {
 	Port                 int            `yaml:"port" mapstructure:"port"`
 	Authenticator        Authenticator  `yaml:"authenticator" mapstructure:",squash"`
 	DisableAutoDiscovery bool           `yaml:"disable_auto_discovery" mapstructure:"-"`
-	EnableDependenciesV2 bool           `yaml:"enable_dependencies_v2" mapstructure:"-"`
```

Comment:
I expect there to be some code that actually reads this field, not seeing anything like that removed in the PR.

Reply:
Did a recursive case-insensitive search in my jaeger root project dir and couldn't find `EnableDependenciesV2` or `enable_dependencies_v2`.

Maybe something wrong with my search, can you see any references?

Reply:
found it - the logic for that flag was already removed in #1364